manager/usermanager/createUser.go

```diff
@@ -26,10 +26,7 @@ func CreateUser(userStore *store.UserStore, user *models.UserCredentials) (*mode
 	}
 
 	var newUser *models.UserCredentials
-	if user.Role == models.RoleAdmin {
-		newUser = user
-		newUser.Password = string(hashedPassword)
-	} else {
+	if isSignup {
 		newUser = &models.UserCredentials{
 			UID:             uuid.Must(uuid.NewRandom()).String(),
 			UserName:        user.UserName,
```

Comment:
What do we set from UI? Username or UnverifiedEmail?

Reply:
UnverifiedEmail is post from UI but we store it under `Username` field as we will have to verify that another user does not exist with the same username already

Reply:
ok